gradle/ci-support.gradle

```diff
@@ -29,6 +29,11 @@ gradle.taskGraph.whenReady {
             subproject.tasks.findByName("javadoc")?.enabled = false
             subproject.tasks.findByName("delombok")?.enabled = false
             subproject.tasks.findByName("japicmp")?.enabled = false
+            subproject.tasks.findByName("spotlessJavaCheck")?.enabled = false
+            subproject.tasks.findByName("spotlessGroovyGradleCheck")?.enabled = false
+            subproject.tasks.findByName("checkstyleMain")?.enabled = false
+            subproject.tasks.findByName("checkstyleTest")?.enabled = false
+            subproject.tasks.findByName("checkstyleJarFileTest")?.enabled = false
```

Comment:
why do we need to disable them here? :)

Reply:
I need to disable the analysis over `jarFileTest` folder. But trying a global config and delete this one

Reply:
Perhaps we could do something like `tasks.ofType(SpotlessCheck.class)` (artificial name) to disable them all in the root build file without listing?